pino-mongodb.js

```diff
@@ -41,8 +42,11 @@ function handleConnection (e, mClient) {
   })
 }
 
+const options = { useNewUrlParser: true }
+if (program.unified) { options.useUnifiedTopology = true }
```

Comment:
Looks like you can avoid creating the new variable altogether! Could you please pass an object with props like before and just change the value depending on the program?

Reply:
That was my first try, but older version of mongo will complain that `useUnifiedTopology` is not supported if is see this option, even if it is `false`, that's why I added this if. I'll try with `undefined` instead of false, let see if it works without warning.

Reply:
I double-checked and mongodb version <= 3.1.13 will output a warning : `the options [useUnifiedTopology] is not supported` when it sees the key in the options object, whether the value is `false` or `undefined`.
So for me, the `if` is required to avoid any warning.

Reply:
Well, alright!